package.json

```diff
@@ -147,7 +147,7 @@
   "detox": {
     "test-runner": "jest",
     "runner-config": "RNTester/e2e/config.json",
-    "specs": "RNTester/e2e",
+    "specs": "",
```

Comment:
Can this be deleted since it's deprecated ?

Reply:
Good question, that's what I first thought as well. However, for backwards-compatibility reasons, they suggest to keep the field set to an empty string. We can only remove it once the library gets updated to `13.0.0` I assume.

There is a whole section with additional information in the [migration guide](https://github.com/wix/Detox/blob/master/docs/Guide.Migration.md#migrating-from-detox-120x-to-121x) about it:

> For the curious ones, who want to know more why we should use an empty string ("") instead of deleting "specs" [...]